dts/bindings/sensor/ti,bq274xx.yaml

```diff
@@ -11,6 +11,19 @@ compatible: "ti,bq274xx"
 include: i2c-device.yaml
 
 properties:
+    part:
+      type: string
+      required: true
+      description: |
+        Part number, note that not all BQ274XX devices are compatible with this
+        driver, such as the BQ27410.
+      enum:
+         - "bq27411"
+         - "bq27421"
+         - "bq27425"
+         - "bq27426"
+         - "bq27441"
```

Comment:
This is not the right approach from a devicetree perspective. The right way to do this is to make a binding for each of these parts, each with its own compatible. You can share common properties using `include` in each of these bindings of a common file.

Then, to keep some common driver code, you can redefine DT_DRV_COMPAT as many times as you need to create specific devices for each part. See drivers/i2c/i2c_tca954x.c for an example of this technique.

Reply:
done